tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -440,44 +357,6 @@ module.exports = {
         })
         .useCss()
     },
-    /**
-     *
-     * @param {string} fileName
-     * @param {string} elementType
-     *
-     * @returns {string}
-     */
-    getFileRowSelectorByFileName: function (fileName, elementType = 'file') {
```

Comment:
I think that should stay in the filesList PO. How can a row find it self? when the row is found by the fileList, it can be initialised and do all its work

Reply:
As far as i understood the objective of splitting into filesRow PO  was to keep all the methods and elements related to file's row in same PO. That is why i did this. So what should i do, should i keep it as it is or move it back to `fileList` PO?